src/main/java/android/util/Base64.java

```diff
@@ -0,0 +1,741 @@
+/*
+ * Copyright (C) 2010 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package android.util;
```

Comment:
It is a 1:1 port of the android framework code, and thus not an extension. It will not override Android platform code, as it is not directly run by an Android package, so it can / should be overridden by the Android local code. This invalidates renaming the package.

Using the implementation from Java 8 requires Java 8, which is the entire reason this change was necessary. Java 7 has a Base64 implementation, but it would not support the requirements for the Java 8 usage in this library, while the Android code is an obvious alternative given the issue originates when using this code with Android.